source/habitat_quality.rst

```diff
@@ -179,51 +179,54 @@ This section outlines the specific data used by the model. Please consult the In
 
-* **Current Land Cover** (required). A GIS raster dataset, with an integer LULC code for each cell. The LULC raster should include the area of interest, as well as a buffer of the width of the greatest maximum threat distance. Otherwise, locations near the edge of the area of interest may have inflated habitat quality scores, because threats outside the area of interested are not properly accounted for. *The LULC codes must match the codes in the "Sensitivity of land cover types to each threat" table below.*
+* **Current Land Cover** (required). A GIS raster dataset, with an integer LULC code for each cell. The LULC raster should include the area of interest for the study, such that the area of interest is inset from the edges of the LULC raster by the width of the greatest maximum threat distance. Otherwise, locations near the edge of the area of interest may have inflated habitat quality scores, because threats outside the area of interest are not properly accounted for. This occurs when threats are at the edge of the LULC raster and the study area of interest is the entire LULC boundary. *The LULC codes must match the codes in the "Sensitivity of land cover types to each threat" table below.*
 
-* **Future Land Cover** (optional).  A GIS raster dataset that represents a future projection of LULC in the landscape with an integer LULC code for each cell. This file should be formatted exactly like the "Current Land Cover" above. LULC classes that appear on both the current and future maps should have the same LULC code.  LULC types unique to the future map should have codes not used in the current LULC map. Again, the LULC raster should include the area of interest, as well as a buffer of the width of the greatest maximum threat distance. Otherwise, locations near the edge of the area of interest may have inflated habitat quality scores, because threats outside the area of interested are not properly accounted for.
+* **Future Land Cover** (optional).  A GIS raster dataset that represents a future scenario of LULC in the landscape with an integer LULC code for each cell. This file should be formatted exactly like the "Current Land Cover" above. LULC classes that appear on both the current and future maps should have the same LULC code.  LULC types unique to the future map should have codes not used in the current LULC map. Again, the LULC raster should include the area of interest for the study, such that the area of interest is inset from the edges of the LULC raster by the width of the greatest maximum threat distance. Otherwise, locations near the edge of the area of interest may have inflated habitat quality scores, because threats outside the area of interest are not properly accounted for. This occurs when threats are at the edge of the LULC raster and the study area of interest is the entire LULC boundary.
```

Comment:
I would be inclined to extract the redundant parts ("Again, the LULC raster should include...") of these raster descriptions. Either leaving it in the first one and letting the others refer up to it, or extracting it from all and placing it in a seperate `.. note::` block, or something similar.